### PR TITLE
Bump mountinfo 0.2.0

### PR DESCRIPTION
**Pull request by kolyshkin**
This should considerably speed up `Mounted()` call (for non-bind mounts and for all mounts on recent kernels).

Also, a wrapper for Mounts() is not removed, as its functionality is now implemented in upstream.

**Note the (potentially) breaking change:**
Previously, calling `Mounted()` with a non-existing path as an argument resulted in an error (wrapped `ENOENT`), while now it will return "not mounted". The change is intentional -- the call is not to check if the path exists, it is to check whether it is mounted.

### CODE REVIEW

**Review by rhatdan — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-go 1.15
+go 1.14
```
Comment (saschagrunert) [inline go.mod:1]: What is the reason for this downgrade?

Comment (rhatdan) [reply go.mod:1]: It is explained in the PR.

**Comment by rhatdan** [comment]
@kolyshkin Could you rebase and we can merge.  This Vendor was done by dependabot.

**Pull request closed without merging**